src/m365/spo/commands/term/term-add.ts

```diff
@@ -242,7 +256,7 @@ class SpoTermAddCommand extends SpoCommand {
         }
 
         const requestOptions: any = {
-          url: `${spoAdminUrl}/_vti_bin/client.svc/ProcessQuery`,
```

Comment:
I'm getting an access denied error here when using the webUrl option. I'm a global admin and a sitecollection admin. I tried a tenant-termset and a sitecol termset. Both give me the same error. 

What about you?

Reply:
Hi @martinlingstuyl 

With Global admin and SiteCollection admin, both the commands works fine for me.

**Scenario 1: Tenant Termset**
![image](https://github.com/pnp/cli-microsoft365/assets/8310319/a1f5f1c2-4ab5-4a26-b590-cad0485c1a17)

Output:
![image](https://github.com/pnp/cli-microsoft365/assets/8310319/02011c72-aea7-4354-93b6-0981d137bc3d)

**Scenario 2: SiteCollection Termset**
![image](https://github.com/pnp/cli-microsoft365/assets/8310319/cedb37ec-a331-4457-8025-01f5566bc3f2)

Output:
![image](https://github.com/pnp/cli-microsoft365/assets/8310319/d0a56d70-9b52-42ba-843e-2dabb57f9a0d)

Can you please re-check?
Thank you.

Reply:
Hmmm... that's odd. Will try it again 👍